singleFrameProcCoreTasks/calibimg.rst

```diff
@@ -0,0 +1,216 @@
+#############
+CalibrateTask
+#############
+
+Given a properly characterized exposure (which means one with a PSF
+determined and shipped along with the image, commonly done previous to
+running this Task by :doc:`CharacterizeImage <charimg>`), detect
+sources, measure their positions, and do a photometric measurement on
+them.
```

Comment:
It's true that measurement is part of this task, but that's different from the astrometric and photometric *calibration*.  That is to say that a major part of this task is to fit models to the object positions to reduce the residual with a reference catalog.